vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -4,34 +4,27 @@ sidebarDepth: 2
-Beginning with Entando 7.1, the `ent bundle` command and its convenience methods introduce a streamlined process to govern the structure, files and management of Entando bundles. The bundle development lifecycle is divided into 6 stages, each corresponding to a subcommand.
+Beginning with Entando 7.1, the `ent bundle` command and its convenience methods introduce a streamlined process to govern the structure, files and management of Entando bundles. The bundle development lifecycle is divided into 6 stages, each corresponding to a subcommand:
+
+- [Initialization](#initialization): `ent bundle init` initializes a new bundle project, either with the default files and folders or from an existing bundle in an Entando Hub. The bundle format relies on a single JSON descriptor as the project manifest. 
 
-The `init` subcommand initializes a new bundle project, either with the default files and folders or from an existing bundle in an Entando hub. The bundle format relies on a single JSON descriptor as the project manifest. 
+- [Build](#build): With the structure established, `ent bundle build` generates micro frontend (MFE) and microservice (MS) components. These are filtered by type and name and assigned version numbers. 
 
-With the structure established, the `build` subcommand executes processes appropriate to component type and stack.Micro frontend (MFE) and microservice (MS) components are generated in parallel. They are filtered by type and name and assigned version numbers. 
+- [Run](#run): `ent bundle run` and Keycloak integration enable components to be tested locally, external to an Entando cluster.   
 
-The `run` subcommand in conjuction with Keycloak integration enables components to be tested locally, external to an Entando cluster. Multiple components are run in parallel via commands dependent on type and stack, with a log file for each individual component generated in the .entando/logs directory.  
+- [Package](#package): `ent bundle pack` constructs the artifacts and Docker images for the bundle. A single image is created for the bundle and all micro frontends while each microservice is allocated its own image. 
 
-The `pack` subcommand constructs the artifacts and Docker images for the bundle. A single image is created for the bundle and all micro frontends while each microservice is allocated its own image. These are published to a Docker repository, after which the bundle can be deployed and installed. 
+- [Publish](#publish): `ent bundle publish` pushes the Docker images to a Docker repository, after which the bundle can be deployed and installed. 
 
-Once a bundle has been published, it can be delivered to the ECR of an Entando Application via the `deploy` subcommand. The bundle custom resource is exported and tags are retrieved from Docker Hub.
+- [Deploy](#deploy): `ent bundle deploy` delivers a published bundle to the ECR of an Entando Application. The bundle custom resource is exported and tags are retrieved from Docker Hub.
```

Comment:
still don't think export is accurate

Reply:
i guess leave a comment on notion. to export a CR is common terminology and logical

Reply:
I don't think export actually works here. What's happening here is creating the descriptor file for a custom resource - generated is the standard language in all the tools, hence ent prj generate-cr.  Export implies taking something out of one system so it can be used in another but the custom resource is actually being written (generated) from the data in the repository. Arguably if the CR existed as a thing in the repository, then it might be an export but that isn't the case here.

Reply:
thanks for clarifying; i had thought that the CR was generated in the cluster before the bundle was pulled into the ECR